rust/src/lib.rs

```diff
@@ -13,13 +14,13 @@ pub use crate::trees::*;
 pub fn find_candidates(
     x: impl Iterator<Item = geo_types::LineString>,
     y: impl Iterator<Item = geo_types::LineString>,
-    dist: f64,
+    distance_tolerance: f64,
     slope_tolerance: f64,
```

Comment:
`angle_tolerance` here, now that it's in units of degrees?

Could be confused with slope as in gradient.

Reply:
Really good point